### PR TITLE
add probe select

### PR DESCRIPTION
**Pull request by freasy**

### CODE REVIEW

File: probes/cmsis-dap.cfg [hunk 1]
```diff
@@ -0,0 +1 @@
+source [find interface/cmsis-dap.cfg
```
Comment (tridge) [inline probes/cmsis-dap.cfg]: missing closing ]

File: esc_unlocker.py [hunk 2]
```diff
@@ -185,6 +197,15 @@ def update_status_led(color):
 root.grid_rowconfigure(5, weight=1)
 root.grid_columnconfigure(0, weight=1)
 root.grid_columnconfigure(1, weight=1)
+root.grid_columnconfigure(2, weight=1)
+root.grid_columnconfigure(3, weight=1)
+
+# Probe selection
+probe_var = tk.StringVar()
+probe_label = ttk.Label(root, text="Select MCU Type:")
```
Comment (tridge) [inline esc_unlocker.py]: label should be "Select Probe:"

**Review by tridge — CHANGES_REQUESTED** [review]
darn good for first python PR !

File: esc_unlocker.py (hunk 2 above)
Comment (freasy) [reply esc_unlocker.py]: fixed

File: probes/cmsis-dap.cfg (hunk 1 above)
Comment (freasy) [reply probes/cmsis-dap.cfg]: fixed

**Review by tridge — APPROVED** [review]

**Pull request merged**